tests/lib/rules/sort-prop-types.js

```diff
@@ -565,6 +565,42 @@ ruleTester.run('sort-prop-types', rule, {
     }]
   }, {
     code: [
+      'const First = (props) => <div />;',
+      'const propTypes = {',
```

Comment:
can you also (just in case) add a test with `export const propTypes = …` to this rule too?